image-layout.md

```diff
@@ -20,6 +22,7 @@ The image layout is as follows:
     - See [blobs](#blobs) section
 - `oci-layout` file
     - It MUST exist
+    - Its name MUST be `oci-layout`
```

Comment:
I think this is already covered by the earlier `- oci-layout file` line.  Note that we have no `Its name MUST be blobs` entry under `- blobs directory` above (for the same reason).